index.html

```diff
@@ -40,41 +42,59 @@
-          <footer>
-            <i class="fa fa-usd"></i>
-            <span>6.45</span>
-          </footer>
-        </section>
-      </section>
-
-      <section class="section" id="page4">
-        <section class="placeholder top left">
-          <header>Beef Steak</header>
-          <section>A cut of meat sliced perpendicular to the muscle fibers, potentially including a bone.</section>
-          <footer>
-            <i class="fa fa-usd"></i>
-            <span>6.45</span>
-          </footer>
-        </section>
-      </section>
+      
 
     </section>
-
+    
+    <!-- JQuery -->
+    <script>
+      //JSON object
+      var x = [ 
+                {
+                  "name":"Fish Fillet",
+                  "description":"Fish fillets comprise the flesh of the fish, which is the skeletal muscles and fat as opposed to the bones and viscera.",
+                  "price":"5.50",
+                  "placeholder":"bottom right",
+                  "url":"img/menu/fish_bg.jpg"
+                },
+                {
+                  "name":"Pizza",
+                  "description":"Oven-baked flat bread generally topped with tomato sauce and cheese. It is commonly supplemented with a selection of meats, vegetables and condiments.",
+                  "price":"6.45",
+                  "placeholder":"top left",
+                  "url":"img/menu/pizza_bg.jpg"
+                },
+                {
+                  "name":"Beef Steak",
+                  "description":"A cut of meat sliced perpendicular to the muscle fibers, potentially including a bone.",
+                  "price":"6.45",
+                  "placeholder":"top left",
+                  "url":"img/menu/steak_bg.jpeg"
+                }
+              ];
+      var pageElement = '<section class="section page">' +
+                            '<section>' +
+                              '<header></header>' +   //name
+                              '<section></section>' + //description
+                              '<footer>' +
+                                '<i class="fa fa-usd"></i>' +
+                                '<span></span>' +     //price
+                              '</footer>' +
+                            '</section>' +
+                          '</section>';
+      
+      //loop
+      x.forEach(function(element, i) {
+        $(pageElement).insertAfter(".section:nth(" + (i) + ")");
+        $(".page:nth(" + i + ")").css({
+                                        "background": "url(" + element.url + ") no-repeat center center",
+                                        "background-size": "cover"
+                                      });
+        $(".page:nth(" + i + ") section:nth(0)").addClass("placeholder " + element.placeholder);
+        $(".page:nth(" + i + ") header").text(element.name);
+        $(".page:nth(" + i + ") section:nth(1)").text(element.description);
```

Comment:
also, you can add classes to all elements where you change text and refer them by class